demos/node/fonts.js

```diff
@@ -0,0 +1,46 @@
+// node fonts.js > output.html
+//
+// This demo shows how to use node-canvas to load a font (woff / otf).
```

Comment:
When run on the command line via node, the ChordSymbol test cases fall back to Times and Arial, because the web fonts are not available. This demo shows how we can register custom fonts like Roboto Slab and PetalumaScript so that they will appear in node-canvas.